pages/api/fetchy-boi/latest-run.ts

```diff
@@ -0,0 +1,10 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getLastestFetchyRun } from "../../../shared/githubDataStore";
+
+export default async function handler(
```

Comment:
Why do we need an API endpoint for this?
Are we going to get the UI call it as well as load it from the props with ssr?

Reply:
Ah will need to double check with @jishaal here - is the plan to have the last checked at datetime update after the page has loaded? If so we need it. My thinking is this endpoint will drive the "new data" ux

Reply:
I think it was for the case where the page is left open / not refreshed and the "last checked" datetime gets stale, solution being polling this API to get latest and render.

In terms of the "new data" ux, would need another API/way to figure out when new data is available as this API wouldn't be sufficient

Reply:
ahh sweet

Reply:
Sweet. Sounds good to me.

I think we could _probably_ extend this API to have the data about last updated too?

Reply:
Sounds good!